src/main/webapp/css/loginpage.css

```diff
@@ -0,0 +1,83 @@
+
+
+body {
+  text-align:center;
+  margin: 0;
+  font-family: sans-serif;
+  line-height: 1.6;
+  font-size: 18px;
+  line-height: 1.6;
+  color: #444;
+  background-color: #eeeeee;
+}
+
+.navbar-brand {
+  padding: 0 15px;
+  height: 200px;
+  line-height: 80px;
+}
+
+nav {
+  background-color: lightblue;
+  line-height: 3;
+}
+
+
+nav b {
+  text-align: center;
+  background-color: darkslategrey;
+  line-height: 8;
+  column-width: 100000px;
+  color: white;
+  display: inline-block;
+  font-size: 24px;
+  text-decoration: none;
+  font-family: monospace;
+  font-weight: bolder;
+}
+
+#navTitle {
+  font-size: 36px;
+}
+
+#container {
+  margin-left: auto;
+  margin-right: auto;
+  width: 800px;
+}
+
+h1 {
+  color: #757575;
+}
+
+input {
+  font-size: 18px;
+  border: none;
+  border-bottom: 2px solid fuchsia;
+  background-color: #eeeeee;
+  outline: none;
+}
+
+p{
+  text-align:center;
+}
+
+button {
+    background-color: darkturquoise;
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    font-size: 20px;
+    border-radius: 16px;
+    -webkit-text-stroke-width: thin;
+}
+
+button {
+    -webkit-transition-duration: 0.4s; /* Safari */
```

Comment:
Might want to consider supporting firefox as well, just to get some understanding how different browsers behave :)